### ISSUE TITLE
Question (discussions)

### ISSUE THREAD

**Issue by SpyC0der77** [TimMcCool/scratchattach#312]
Can you enable discussions on this repo?

**Comment by FAReTek1** [comment]
might try later, why?

**Comment by SpyC0der77** [comment]
So that I can ask for feedback on something I am making (ScratchAnalytics)

**Comment by DagriffpatchfanHosting** [comment]
We are essentially using issues as a place to discuss.

**Comment by SpyC0der77** [comment]
Discussions are better than issues

**Comment by DagriffpatchfanHosting** [comment]
> Discussions are better than issues

True, but everyone is so used to using issues that switching over might be hard

**Comment by SpyC0der77** [comment]
We don't have to switch over, there aren't really any discussions in issues rn

**Comment by SpyC0der77** [comment]
We would just be adding a better feature

**Comment by DagriffpatchfanHosting** [comment]
> We would just be adding a better feature
> If nobody uses a feature, who cares if we deprecate it?

You mean add it?

**Comment by FAReTek1** [comment]
afaik, onky timmccool can add this. so you may have to wait a while/it might not happen

**Comment by DagriffpatchfanHosting** [comment]
> afaik, only timmccool can add this. so you may have to wait a while/it might not happen

@TimMcCool  doesn’t check this repo very often :(

**Comment by TimMcCool** [comment]
okay

**Comment by SpyC0der77** [comment]
you turned it on?

**Comment by SpyC0der77** [comment]
I dont see discussions

**Comment by SpyC0der77** [comment]
@TimMcCool

**Comment by TimMcCool** [comment]
It's turned on now
